dashboard/src/modules/components/DatePickerComponent/index.jsx

```diff
@@ -29,12 +33,13 @@ function DatePickerWidget({
       if (date > toDate) {
         date.setDate(date.getDate() + 1);
```

Comment:
Could we consider all the UTC vs local time date work for a follow-up PR?

Reply:
Not really.  This PR is providing the connection between UI elements (which produce local time values) and the Server database (which provides UTC time values), so the UTC vs. local time is central to this work.

If we want to split them, it would be better to provide the UTC time support as a **precursor** PR.  (And, I don't think that taking that approach will get this PR in faster.)

Reply:
Why can't we fix this after merging the PR?

Can we survive with the time handling the way it is?  I think so.  We need to consider allowing work to go forward and improving it later.

Reply:
I don't be this is "central" to the work.  Time handling can be done as a separate follow-on PR.

Reply:
I'm willing to sign on to deferring work from this PR, provided we start by removing the Patternfly `DatePicker`, since that's where all the time handing problem originate.  When that goes, the time-based selection feature will go to, but perhaps that acceptable for the moment?

"Can we survive?" is a hard question to answer...how much technical debt can we take on before it crushes us?